docs/recipes/code-coverage.md

```diff
@@ -170,6 +170,28 @@ after_success:
 
 Your coverage report will then appear on coveralls shortly after Travis completes.
 
+## Integrating with AVA's Watch mode
+
+While AVA's watch mode [won't](https://github.com/avajs/ava/issues/1100) integrate with `nyc` directly out of the box, you can achieve that using [nodemon](https://github.com/remy/nodemon).
+
+First, install `nodemon` as a dev dependency:
+
+```bash
```

Comment:
Done, good spot. I remember some of my editors had issues in the past recognising `console` but it's probably an old reflex. Thank you. Please review.